src/app/components/top-bar/top-bar.scss

```diff
@@ -1,5 +1,9 @@
 @import '../../styles/base';
 
+.material-icons.md-dark {
```

Comment:
this is not really ok since it will affect all the app, you should move it inside `.top-bar` definition to affect only top-bar component. Maybe you can use a mixin for this.

Reply:
Also, avoid these kind of rules, they apply globally and it's very hard to track down in case of a bug. A component should not affect global styling.